### DISCUSSION TITLE
ZgBridge with Neo NAS-AB02B0 siren

### DISCUSSION THREAD

**Discussion by dumi-cz** [arendst/Tasmota#11565] [Ideas]
Hi, is it possible to use NAS-AB02B0 siren with tasmotized ZgBridge?

In zigbee2mqqt is supported. 

Do you plan to add support for it to tasmota?

Thanks

**Comment by Jason2866** [comment]
Probably it is working with Tasmota https://zigbee.blakadder.com/Neo_NAS-AB02B0.html

**Reply by dumi-cz** [reply]
I know this page. 

I paired the siren with my tasmotized ZgBridge, but i dont know ho to activate sound, led flasher or change volume/melody.

I experimenting with zbsend command a lot, but without success :(

**Reply by dumi-cz** [reply]
I tried to investigate zigbee2mqtt source code.

I found this:
https://github.com/zigpy/zha-device-handlers/blob/dev/zhaquirks/tuya/siren.py

there is some cluster number 0xEF00 and some attributes numbers, but i don't know, ho to use them :(

**Reply by dumi-cz** [reply]
as i look to zigbee2mqtt, some zigbee converters, the command to sound the siren should be:
zbSend { "device" : "0x8D49" , "send" : "EF00_00/00560168000101"}

but it doesn't work.

Siren only flashing rapidly after powering up.

**Reply by rebel7580** [reply]
Tasmota Zigbee doc says use "!" instead of "_" for cluster-specific commands. Maybe try that?  Just a wild (uneducated) guess on my part though!!!

**Reply by dumi-cz** [reply]
Thanks, i try that already. without success :(

**Comment by dumi-cz** [comment]
I found another source code, but i can't properly format Zsbend command to sound the siren.
Can someone help, pls?

https://github.com/stas-demydiuk/domoticz-zigbee2mqtt-plugin/issues/389

**Comment by dumi-cz** [comment]
From source code of zigbee2mqqt and other opensource project i found that sending payload of 00106801000101 with command 00 to cluster EF00 will activate the siren, but nothing happens :(
logs from zibgee bridge:

11:41:15.981 CMD: zbsend {"device" : "0x8D49" , "send" : "EF00!00/00516801000101"}
11:41:15.983 SRC: WebConsole from 192.168.1.10
11:41:15.985 CMD: Group 0, Index 1, Command "ZBSEND", Data "{"device" : "0x8D49" , "send" : "EF00!00/00516801000101"}"
11:41:15.987 ZIG: guessing endpoint 1
11:41:15.989 ZigbeeZCLSend device: 0x8D49, group: 0x0000, endpoint:1, cluster:0xEF00, cmd:0x00, send:"00516801000101"
11:41:15.990 ZbSend: shortaddr 0x8D49, groupaddr 0x0000, cluster 0xEF00, endpoint 0x01, cmd 0x00, data 
11:41:15.992 ZIG: ZbEZSPSend 340000498D040100EF01014001000042010A01420000516801000101
11:41:15.995 MQT: stat/tasmota_ZBBridge/RESULT = {"ZbSend":"Done"}
11:41:16.015 ZIG: {"ZbEZSPReceived":"3400004A"}
11:41:16.234 ZIG: {"ZbEZSPReceived":"450000040100EF01014001000071FFDE498DFFFF0508420B000002"}
11:41:16.236 ZIG: {"ZbZCLReceived":{"groupid":0,"clusterid":"0xEF00","srcaddr":"0x8D49","srcendpoint":1,"dstendpoint":1,"wasbroadcast":0,"LinkQuality":139,"securityuse":0,"seqnumber":113,"fc":"0x08","frametype":0,"direction":1,"disableresp":0,"manuf":"0x0000","transact":66,"cmdid":"0x0B","payload":"0000"}}
11:41:16.245 MQT: tele/tasmota_ZBBridge/RESULT = {"ZbResponse":{"Device":"0x8D49","Command":"EF00!00","Status":0,"StatusMessage":"SUCCESS","Endpoint":1,"LinkQuality":139}}
11:41:16.249 ZIG: {"ZbEZSPReceived":"3F0000498D040100EF0101400100004A010000"}

**Comment by dumi-cz** [comment]
nobody use this siren?

Is there someone ho could help me to debug this problem?

**Comment by dumi-cz** [comment]
Some time have passed.

Still trying to work with the siren but without success :(

Is there anybody who can
